examples/complete/main.tf

```diff
@@ -21,7 +21,7 @@ module "vpc" {
 # Security group
 ###########################
 module "sg" {
-  source  = "terraform-aws-modules/security-group/aws//modules/redshift"
+  source  = "terraform-aws-modules/security-group/aws/modules/redshift"
```

Comment:
`//` was correct (in theory).

Ref: https://www.terraform.io/docs/language/modules/sources.html#hashicorp-consul-aws-modules-consul-cluster

Reply:
oh, weird. when I ran the pre-commit check locally, it errored on it